main_flex.go

```diff
@@ -6,9 +6,12 @@ import (
 	"github.com/rivo/tview"
 )
 
+var guildsVisible bool = true
```

Comment:
This is not necessary, you can easily check whether the guilds tree is visible or not using `(*tview.Flex).GetItem` or `(*tview.Flex).GetItemCount`.